wasm_src/ast_wrapper/ast_wrapper.cc

```diff
@@ -52,6 +52,14 @@ EMSCRIPTEN_KEEPALIVE AstFrameSet* getFrameFromFitsChan(AstFitsChan* fitschan)
         return nullptr;
     }
 
+    // work around for missing CTYPE1 & CTYPE2
+    if (checkSkyDomain) {
+        const char *domain = astGetC(frameSet, "Domain");
+        if (!strstr(domain, "SKY")) {
+            return nullptr;
```

Comment:
if we're returning `nullptr` here, we should call `astDelete` first, shouldn't we? otherwise we're leakingWebAssembly  memory